CONTRIBUTING.md

```diff
@@ -8,59 +8,40 @@ We tag issues that are good candidates for those new to the code with [`good fir
 
 ## Follow the coding style
 
-The `.eslintrc` file defines all. We use [JSDoc](http://usejsdoc.org/) with [TypeScript](https://github.com/Microsoft/TypeScript/wiki/JSDoc-support-in-JavaScript). Annotations are encouraged for all contributions.
-
-## Pull request titles
```

Comment:
this, CLA, and "Adding Images to a Readme" sections were just moved down the page

Reply:
Thanks for help